traveler-ios-ui/TravelerKitUI/ViewControllers/BookingQuestionsViewController.swift

```diff
@@ -154,20 +154,30 @@ extension BookingQuestionsViewController: FormViewDelegate {
         switch errors?.first {
         case .none:
             delegate?.bookingQuestionsViewControllerDidCheckout(self)
-        case .some(.invalidAnswer(let groupIndex, let questionIndex, _)):
-            self.error = errors?.first
-
-            let indexPath = IndexPath(item: questionIndex, section: groupIndex)
-            formView.reloadFields(at: [indexPath])
-            formView.scrollToField(at: indexPath, animated: true)
+        case .some(.invalidAnswer):
+            if let errors = errors {
```

Comment:
This is an incorrect pattern. Youre switching from `errors.first` and then catching ALL errors